### DISCUSSION TITLE
DateTime timezone

### DISCUSSION THREAD

**Discussion by carlosfiori** [prisma/prisma#4153] [Q&A] [answered]
Currently, with a field of type DateTime and @default(now()), when creating a new record this field is saved with a date on UTC timezone, is it possible to change this default timezone?

**Comment by ryands17** [comment, marked as answer]
Hey @carlosfiori 👋 
That is currently not possible right now. You can follow [this](https://github.com/prisma/prisma/issues/3292) request for more info. As a workaround, the best way would be to convert to the required timezone locally.

**Reply by carlosfiori** [reply]
Thanks

**Comment by AmeerMuaviya** [comment]
In my case i fetched datetime as it is from database and used `new Date(date_from_db)` which changed utc time format according to my system

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
